### ISSUE TITLE
[v2]What is the minimum Kubernetes version required to use keda?  

### ISSUE THREAD

**Issue by silenceper** [kedacore/keda#1160]
Which place clearly indicates the minimum required Kubernetes version?

I think if it is marked in the Readme, it will be very useful for people who are new to this project.

**Comment by silenceper** [comment]
mybe v1.16+

**Comment by zroubalik** [comment]
I think you should be able to deploy KEDA on v1.13+

If we change CRDs apiversion to `v1`, it will be v1.16+

**Comment by silenceper** [comment]
```go
kubectl apply -f https://github.com/kedacore/keda/releases/download/v2.0.0-beta/keda-2.0.0-beta.yaml
```
I failed to deploy on the 1.15 version and succeeded on the 1.16 cluster.

Because OpenAPI  `x-kubernetes-list-map-keys` was add in 1.16.

**Comment by silenceper** [comment]
I think it is necessary to add some compatibility tests for the k8s version and inform users of the minimum version requirements of k8s.

**Comment by zroubalik** [comment]
OK, thanks for the confirmation. We should add info on this to the documentation.

**Comment by silenceper** [comment]
fixed in #1202
